quarkus/server/pom.xml

```diff
@@ -77,5 +77,25 @@
                 </dependency>
             </dependencies>
         </profile>
+
+        <profile>
+            <id>product</id>
+            <dependencies>
+                <dependency>
+                    <groupId>org.keycloak</groupId>
+                    <artifactId>keycloak-quarkus-server</artifactId>
+                    <exclusions>
+                        <exclusion>
+                            <groupId>io.quarkus</groupId>
+                            <artifactId>quarkus-logging-gelf</artifactId>
+                        </exclusion>
+                        <exclusion>
+                            <groupId>io.quarkus</groupId>
+                            <artifactId>quarkus-logging-gelf-deployment</artifactId>
+                        </exclusion>
```

Comment:
One last thing I noticed (sorry, didn't notice it before).

Wouldn't it make more sense to exclude it in `runtime` and `deployment` modules respectively, rather than here and in tests?

Reply:
@vmuzikar That was made in the first version of this PR as these dependencies were marked as `provided`, so not included in the distribution. How exactly would you exclude it from the `runtime`/`deployment` modules? 

I'm not aware of any other option than the mentioned and exclusion from the "referencer" pom (quarkus/server). I'm not sure what might be the better approach for this; I don't see any problem in either of these approaches. 

The thing is that these transitive deps should be included for the community by default. I don't want to provide any "negative" property activation for these profiles.

Reply:
I was thinking about it and I think we can't avoid a property activated profile. Created https://github.com/mabartos/keycloak/pull/262 for that. We should also use the same approach for Oracle in the 22 branch.

Also in that PR I tweaked a bit how the tests detect GELF. It might happen that in testsuite GELF won't be present on the classpath but it will be in server and we'd accidentally skip the tests.

Let me know, what you think.

We'll also need to update the [docs](https://www.keycloak.org/server/logging#_centralized_logging_using_gelf) to exclude GELF on `-Dproduct`.

Reply:
@vmuzikar It's probably a better approach to include these deps in an additive manner than exclude them from parent pom. And for that purpose, there's no better way than providing the `-Dproduct` property. 

@vmuzikar Thanks, I'll include your changes.